samples/boards/stm32/mco/boards/stm32f746g_disco.overlay

```diff
@@ -0,0 +1,44 @@
+/* The clock that is output must be enabled. */
+&clk_lse {
+	status = "okay";
+};
+
+&clk_hse {
+	status = "okay";
+};
+
+/* See reference manual (RM0385):
+ *   01: LSE oscillator selected
+ */
+#define MCO1_SEL_LSE 1
+
+/* See reference manual (RM0385):
+ *   100: division by 2
+ */
+#define MCO1_PRE_DIV_2 4
+
+&mco1 {
+	status = "okay";
+	clocks = <&rcc STM32_SRC_LSE MCO1_SEL(MCO1_SEL_LSE)>;
+	prescaler = <MCO1_PRE(MCO1_PRE_DIV_2)>;
+	pinctrl-0 = <&rcc_mco_1_pa8>; /* D10 (CN7) */
+	pinctrl-names = "default";
+};
+
+/* See reference manual (RM0385):
+ *   10: HSE oscillator clock selected
+ */
+#define MCO2_SEL_HSE 2
+
+/* See reference manual (RM0385):
+ *   111: division by 5
+ */
+#define MCO2_PRE_DIV_NONE 7
```

Comment:
Why is the macro named `DIV_NONE` if this selects `prescaler = 5`?

Reply:
Because I changed my mind and didn't update it properly. (debugging something that didn't work and wanted to see if the frequency was too high for the pin or my logic analyzer).